dropwizard-logging/src/main/java/io/dropwizard/logging/AbstractAppenderFactory.java

```diff
@@ -61,18 +68,42 @@
  *     </tr>
  * </table>
  */
-public abstract class AbstractAppenderFactory implements AppenderFactory {
+public abstract class AbstractAppenderFactory<E extends DeferredProcessingAware> implements AppenderFactory<E> {
+
     @NotNull
     protected Level threshold = Level.ALL;
 
     protected String logFormat;
 
+    @NotNull
+    protected TimeZone timeZone = TimeZone.getTimeZone("UTC");
+
     @Min(1)
     @Max(Integer.MAX_VALUE)
     private int queueSize = AsyncAppenderBase.DEFAULT_QUEUE_SIZE;
 
     private int discardingThreshold = -1;
 
+    @JsonProperty
+    public String getLogFormat() {
```

Comment:
We already have a getter and setter for `logFormat`. I believe moving them higher doesn't provide an additional value.